tests-acceptance/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "test-acceptance",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "test-acceptance": "server-test 'npx vite --port 3000' localhost:3000 cy:run",
```

```suggestion
    "test-acceptance": "server-test 'nom run vite --port 3000' localhost:3000 cy:run",
```
vite is already installed as dependency

---

`npm run` only runs scripts defined in `package.json`. `npx` allows you to run scripts installed by the dependencies you have installed.